### PR TITLE
[WIP] Enable ESLint typescript parsing

### PR DESCRIPTION
**Pull request by batmat**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,3 +17,13 @@ fix-formatting:
 publish:
 	$(MAKE) -C distribution $@
 	$(MAKE) -C services $@
+
+update-package-locks:
+	# Sigh, the sed for forcing https below is recommended by the npm registry team itself...
```
Comment (rtyler) [inline Makefile]: gross!

File: distribution/client/src/lib/update.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ export interface FileOptions {
   flag?: string,
 };
 
+interface Foo {}
```
Comment (rtyler) [inline distribution/client/src/lib/update.ts]: I assume that this is here just to fail the build on purpose?

Comment (batmat) [reply distribution/client/src/lib/update.ts]: yes, but does not work, hence the WIP state :cry:

**Comment by batmat** [comment]
Closing for now.
Anyway, I'm leaning towards trying TSLint instead since it makes more sense to go the typescript direction, instead of trying to refuse ESLint for Typescript.

**Pull request closed without merging**